modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -44,6 +44,13 @@ def bgs_service
         end
 
         def evss_docs_service
+          evss_auth_headers = {}
+          evss_auth_headers['LASTNAME'] = target_veteran.last_name
+          evss_auth_headers['FIRSTNAME'] = target_veteran.first_name
+          evss_auth_headers['EDIPI'] = target_veteran.edipi
+          evss_auth_headers['AUTH_TIMESTAMP'] = Time.zone.now
+
+          auth_headers.merge(evss_auth_headers)
```

Comment:
One thing to keep in mind, is that we probably only want to do this if we're in a CCG context.

I'm pretty sure you should be able to check if we're in a CCG context by doing this ::
```
if @is_valid_ccg_flow
  # logic for appending auth headers
end
```